client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -194,6 +224,39 @@ const PlanOnlyThankYou = ( {
 		},
 	} );
 
+	let upsellProps;
+	if ( promotePlanData ) {
+		upsellProps = {
+			title: translate( 'Advanced tools' ),
```

Comment:
I wonder if we shouldn't add a comment to explain to translators what _tools_ refer to.